public/viewtopic.php

```diff
@@ -150,114 +149,43 @@
         echo "</div>";
         echo "</div>";
 
-        echo "<div class='table-wrapper'>";
-        echo "<table class='table-highlight'><tbody>";
-        // Output all topics, and offer 'prev/next page'
-        foreach ($commentList as $commentData) {
-            // Output one forum, then loop
+        echo "<div class='mb-4'>";
+        // Output all posts, and offer 'prev/next page'
+        foreach ($commentList as $index => $commentData) {
             $nextCommentID = $commentData['ID'];
             $nextCommentPayload = $commentData['Payload'];
             $nextCommentAuthor = $commentData['Author'];
-            $nextCommentAuthorID = $commentData['AuthorID'];
-            $nextCommentDateCreated = $commentData['DateCreated'];
-            $nextCommentDateModified = $commentData['DateModified'];
-            $nextCommentAuthorised = $commentData['Authorised'];
-
-            if ($nextCommentDateCreated !== null) {
-                $nextCommentDateCreatedNiceDate = date("d M, Y H:i", strtotime($nextCommentDateCreated));
-            } else {
-                $nextCommentDateCreatedNiceDate = "None";
-            }
-
-            if ($nextCommentDateModified !== null) {
-                $nextCommentDateModifiedNiceDate = date("d M, Y H:i", strtotime($nextCommentDateModified));
-            } else {
-                $nextCommentDateModifiedNiceDate = "None";
-            }
-
-            sanitize_outputs(
-                $nextCommentPayload,
-                $nextCommentAuthor,
-            );
-
-            $showDisclaimer = false;
-            $showAuthoriseTools = false;
-
-            if ($nextCommentAuthorised == 0) {
-                // Allow, only if this is MY comment (disclaimer: unofficial), or if I'm admin (disclaimer: unofficial, verify user?)
-                if ($permissions >= Permissions::Admin) {
-                    // Allow with disclaimer
-                    $showDisclaimer = true;
-                    $showAuthoriseTools = true;
-                } elseif ($nextCommentAuthor == $user) {
-                    // Allow with disclaimer
-                    $showDisclaimer = true;
-                } else {
-                    continue;    // Ignore this comment for the rest
```

Comment:
You're no longer excluding unauthorized posts (they should only be visible for the author and admins).

Reply:
Fixing this will probably affect the numbering on the links.